pkg/api/types.go

```diff
@@ -1212,6 +1215,23 @@ type StorageOSPersistentVolumeSource struct {
 	SecretRef *ObjectReference
 }
 
+// HugePagesSource represents Linux HugeTlbPage https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt
+type HugePagesVolumeSource struct {
+	// Defaults to 2M
+	// +optional
+	PageSize string
+	// Defaults to PageSize
```

Comment:
I wanted to stick as close to hugetlbfs mount options as possible so think it should default to nothing. Defaulting to PageSize is not a good idea :smile: